### PR TITLE
Menu Item and Change Title

### PR DESCRIPTION
**Pull request by kevinhouyang**
_These changes are being submitted as part of my summer internship application_

I completed two of the recommended changes to the High Fidelity Interface:
- Create a new menu item to trigger some existing application behavior
- Change the title of the Interface window

The first menu item retrieves the sample sound audio clip and plays it from the avatar's current position. The second menu item sets the title of the Interface window.

Please let me know about any questions and concerns! Thank you very much for your consideration!

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by SamGondelman** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15076/HighFidelity-Beta-PR14387-b4b2dfa.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15076/HighFidelity-Beta-PR14387-b4b2dfa.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15076/HighFidelity-Beta-Interface-PR14387-b4b2dfa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15076/HighFidelity-Beta-PR14387-b4b2dfa.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15076/HighFidelity-Beta-Interface-PR14387-b4b2dfa.exe

**Comment by kevinhouyang** [comment]
After building with Xcode, some files are larger than Github's 100MB limit. Any suggestions for pushing them? 

I've just pushed the project file that can be opened and built from Xcode, if that's helpful.

**Comment by birarda** [comment]
@kevinhouyang the diff is pretty hard to parse here, since it includes a bunch of files in `build` that should be gitignored. 

I'm guessing you removed that to commit the Xcode project?

No need to commit the Xcode project - developers locally use CMake to generate whatever files they need to build the project. Can you re-ignore the build directory and make sure those changes are not included in this PR (ideally they would not be in the history either which you could accomplish by amending your current commit).

**Comment by kevinhouyang** [comment]
@birarda thanks for the clarification! Yeah, I pushed the build directory just to show that I had built the project from my end. I re-ignored the build directory, hopefully that cleans things up!

**Comment by birarda** [comment]
Thanks @kevinhouyang! Closing this after reviewing the diff and seeing the successful build.
